src/core/Directus/Services/AssetService.php

```diff
@@ -143,15 +120,14 @@ public function getThumbnail($params)
             }
             $result['mimeType']=$this->getThumbnailMimeType($this->thumbnailDir,$this->fileName);
             $result['file']=$image;
-            $result['filename']=$this->fileName;
+            $result['fileNameDownlaod']=$this->fileNameDownlaod;
```

Comment:
Why is `$result['fileNameDownload']` in camelCase? It's `filename_download` everywhere else.

Reply:
Also you have a type in download; ~downlaod~ -> download